graphene/core/schema.py

```diff
@@ -77,6 +77,7 @@ def schema(self):
             self,
             query=self.T(self.query),
             mutation=self.T(self.mutation),
+            types=[self.T(_type) for _type in list(self._types_names.values())],
```

Comment:
Why do you need to cast `self._types_names.values()` to a `list`? `dict.values()` is already a list ...